lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -124,6 +124,7 @@ module ConfigConstants
 
       DEFAULT_METADATA_AGENT_URL =
         'http://local-metadata-agent.stackdriver.com:8000'.freeze
+      METADATA_AGENT_URL_ENV_VAR_NAME = 'STACKDRIVER_METADATA_AGENT_URL'.freeze
```

Comment:
[Optional] The `_NAME` suffix seems redundant.

Reply:
Done.